app/src/main/java/tech/ula/utils/AndroidUtility.kt

```diff
@@ -387,13 +389,42 @@ class LocalFileLocator(private val applicationFilesDir: String, private val reso
 }
 
 class DeviceDimensions {
-    private var width = 720
-    private var height = 1480
+    private var height = 720
+    private var width = 1480
 
-    fun getDeviceDimensions(windowManager: WindowManager, displayMetrics: DisplayMetrics) {
-        windowManager.defaultDisplay.getMetrics(displayMetrics)
-        width = displayMetrics.widthPixels
+    companion object {
```

Comment:
It looks to me like this can just be private member values, since they aren't being accessed statically anywhere.

Reply:
I'll just move these into the function since they're only used once